Makefile

```diff
@@ -83,3 +81,57 @@ push-check: generate build unit-tests
 	    exit 1; \
 	fi
 	@echo "All looks good - congratulations"
+
+CONTAINER_EXECUTABLE ?= podman
+
+.PHONY: db-tests-prune
+db-tests-prune:
+	-$(CONTAINER_EXECUTABLE) stop image-builder-test-db
+	-$(CONTAINER_EXECUTABLE) rm image-builder-test-db
+
+CHECK_DB_PORT_READY=$(CONTAINER_EXECUTABLE) exec image-builder-test-db pg_isready -d imagebuilder
+CHECK_DB_UP=$(CONTAINER_EXECUTABLE) exec image-builder-test-db psql -U postgres -d imagebuilder -c "SELECT 1"
+
+export POSTGRES_USER=postgres
+export POSTGRES_PASSWORD=foobar
+export POSTGRES_DB=imagebuilder
+
+export PGPASSWORD=$(POSTGRES_PASSWORD)
+export PGDATABASE=$(POSTGRES_DB)
+export PGUSER=$(POSTGRES_USER)
+
+.PHONY: db-tests
+db-tests: dev-prerequisites
+	-$(CONTAINER_EXECUTABLE) stop image-builder-test-db 2>/dev/null || echo "DB already stopped"
+	-$(CONTAINER_EXECUTABLE) rm image-builder-test-db 2>/dev/null || echo "DB already removed"
+	$(CONTAINER_EXECUTABLE) run -d \
+      --name image-builder-test-db \
+      --env POSTGRES_USER \
+      --env POSTGRES_PASSWORD \
+      --env POSTGRES_DB \
+      --publish :5432 \
+      postgres:12
```

Comment:
Idea: if there is a way to provide `fsync=off` configuration to postgres, this significantly increases performance for any kind of testing.

Reply:
although the tests are really quick - I'll definitely try this!

Reply:
Just for the record, I did mean an option in `postgresql.conf` not sure it is ever exposed for this container:

https://www.postgresql.org/docs/current/runtime-config-wal.html#GUC-FSYNC

Reply:
I tried it in #1401. you can set this in `postgresql.conf` or pass as `-c` command line argument
https://github.com/osbuild/image-builder/pull/1401/commits/dfa1096d61e3b2288c49fa6213d0ecd60bb4e89f
but after postgres 13 (I think) the behavior changed and impacts this special case where I try to "test vacuum"
but providing `-c fsync=off` seems to change behavior in general.
Just does not work for the tests I'm intending to run
If it turns out that we can't really test our "vacuum implementation" I'll re-enable and check again if it's faster

Reply:
No need to spend much time on the `fsync` thing it was just a minor idea. Saw the cleanup PR looks great.